docs/_includes/listbox-button.html

```diff
@@ -361,6 +361,114 @@ <h3 id="listbox-button-selected">Selected Listbox Button</h3>
 </span>
     {% endhighlight %}
 
+    <h3 id="listbox-button-unselected">Listbox Button with option subtitles</h3>
+    <p>You can also add subtitles to each option by using <span class="highlight">.listbox-button__subtitle</span></p>
+    <p>In order to make the subtitles accessible, you need to add a clipped punctuation marker. This ensures screen readers will read the subtitle as a separate sentence.</p>
```

Comment:
Did you test it without punctuation first?

Reply:
Yes, a quick test in screen reader shows that the punctuation is definitely a better experience.